src/main/java/com/mraof/minestuck/alchemy/GristGutter.java

```diff
@@ -0,0 +1,116 @@
+package com.mraof.minestuck.alchemy;
+
+import com.mraof.minestuck.skaianet.Session;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.bernie.shadowed.eliotlash.mclib.math.functions.limit.Min;
+
+import java.util.Map;
+
+/**
+ * A class that handles Grist overflow whenever you aqcuire too much grist.
+ * @author Doro
+ */
+public class GristGutter extends GristSet
+{
+	private static Session session;
+	public int gutterTotal = -1;
+	public static final int GUTTER_CAPACITY = 10000;
+	private static final Logger LOGGER = LogManager.getLogger();
+	public void setSession(Session session)
+	{
+		this.session = session;
+	}
```

Comment:
It doesnt seem appropriate for the GristGutter to store a Session in it. Methods in GristGutter that reference session should get the Session from another source. 

getGutterCapacity() should be in Session